administration/src/bp-modules/self-service/hooks/__tests__/useCardGeneratorSelfService.test.tsx

```diff
@@ -50,10 +50,7 @@ describe('useCardGeneratorSelfService', () => {
     expect(result.current.selfServiceCard).toEqual(exampleCard)
     await act(async () => result.current.generateCards())
     expect(toasterSpy).not.toHaveBeenCalled()
-    expect(result.current.selfServiceState).toBe(CardSelfServiceStep.information)
-    expect(result.current.deepLink).toBe(
-      'koblenzpass://koblenz.sozialpass.app/activation/code#ClcKLQoNS2FybGEgS29ibGVuehDmnwEaGAoCCF8SBAjqvgEqBAiLmgEyBgoEMTIzSxIQL%2Fle3xYGNIKS50god4ITmxoUOUYGq%2FjAE99bK4edMPo2I3ojr78%3D/'
-    )
+    expect(result.current.cardGenerationStep).toBe('information')
```

Comment:
hm i think if the check for the correct deeplink is removed, there should be a separate test for that case

Reply:
did you add sth? i couldn't see an update

Reply:
See `administration/src/util/__tests__/getCustomDeepLinkFromQrCode.test.ts`